src/index.js

```diff
@@ -7,7 +7,7 @@ import 'bootstrap/dist/css/bootstrap.min.css';
 import axios from 'axios';
 import {BrowserRouter} from 'react-router-dom';
 
-require('dotenv').config();
+require("dotenv").config();
```

Comment:
Is this consistent? The project should use single or double quotes. I don't care which one it should just be consistent.